lib/arclight/custom_document.rb

```diff
@@ -45,6 +45,7 @@ def to_solr(solr_doc = {})
       Solrizer.insert_field(solr_doc, 'date_range', formatted_unitdate_for_range, :facetable)
       Solrizer.insert_field(solr_doc, 'access_subjects', access_subjects, :facetable)
       Solrizer.insert_field(solr_doc, 'all_subjects', all_subjects, :symbol)
+      Solrizer.insert_field(solr_doc, 'has_online_content', online_content?, :displayable)
```

Comment:
I added e38fd6f92a8e47242a78bdbf42f7359a7801a810 in response to this.  Happy to kill that and generate a `rubocop_todo.yml` file (or put it in the config, but it seems like we would have to exclude the file entirely).